### ISSUE TITLE
On JRuby, initialize Nokogiri::HTML::SAX::PushParser failed with error "initialize_native"

### ISSUE THREAD

**Issue by siuying** [sparklemotion/nokogiri#1147]
with nokogiri "1.6.3.1-java", when i try to initialize PushParser, I get the error:

```
jruby-1.7.13 :001 > require 'nokogiri'
jruby-1.7.13 :002 > parser = Nokogiri::HTML::SAX::PushParser.new
NoMethodError: undefined method `initialize_native' for #<Nokogiri::HTML::SAX::PushParser:0xffcf7b3>
```

Initialize XML PushParser works:

```
jruby-1.7.13 :006 > parser = Nokogiri::XML::SAX::PushParser.new
 => #<Nokogiri::XML::SAX::PushParser:0x79906abc @sax_parser=#<Nokogiri::XML::SAX::Parser:0x784f9afd @warned=false, @document=#<Nokogiri::XML::SAX::Document:0x324895c0>, @encoding="UTF-8">, @document=#<Nokogiri::XML::SAX::Document:0x324895c0>, @encoding="UTF-8"> 
```

**Comment by flavorjones** [comment]
Merged #1148 to fix this, will be in 1.6.6.

**Comment by siuying** [comment]
Thank you @flavorjones @esse !!
